spec/components/searchworks4/availability_component_spec.rb

```diff
@@ -36,7 +36,6 @@
 
       aggregate_failures do
         expect(page).to have_css('.availability-component', text: 'Green Library')
-        expect(page).to have_link 'Stacks', href: "/view/10678312/GRE-STACKS/stackmap"
```

Comment:
Why is this test removed?

Reply:
Because Darcy doesn't want to show the location link at the top level.

Reply:
Its hard to understand the changes without having a written ticket or explanation of what is changing and why.

Reply:
I'm not sure I understand the suggestion. `AccessPanels::LibraryLocationComponent` is used in the sidebar on the catalog#show page, not here.

Reply:
Yeah, I'm a bit confused on this PR.  There's a lot here.